.vscode/tasks.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
a bunch of these vscode files look specific to your dev environment

Reply:
these vscode files are necessary to run LVA! (the tasks.json and launch.json files specifically)

Reply:
why do they have to live in the root of the project is the main concern? they're specific to the lva sample so presumably that's where they should live